tests/sim/conversion_proxy.rs

```diff
@@ -0,0 +1,258 @@
+use crate::utils::*;
+use conversion_proxy::ConversionProxyContract;
+use mocks::fpo_oracle_mock::FPOContractContract;
+use near_sdk::json_types::{U128, U64};
+use near_sdk::Balance;
+use near_sdk_sim::init_simulator;
+use near_sdk_sim::runtime::GenesisConfig;
+use near_sdk_sim::ContractAccount;
+use near_sdk_sim::UserAccount;
+use near_sdk_sim::{call, deploy, lazy_static_include, to_yocto};
+use std::convert::TryInto;
+use std::str;
+
+near_sdk::setup_alloc!();
+
+const PROXY_ID: &str = "request_proxy";
+lazy_static_include::lazy_static_include_bytes! {
+   REQUEST_PROXY_BYTES => "out/conversion_proxy.wasm"
+}
```

Comment:
Now that these tests are split up, I think the names could be simplified


```suggestion
const PROXY_ID: &str = "conversion_proxy";
lazy_static_include::lazy_static_include_bytes! {
   PROXY_BYTES => "out/conversion_proxy.wasm"
}
```

Reply:
Thanks, I also adapted the proxy contract variable name accordingly.